### ISSUE TITLE
NotFoundError with tf.add_check_numerics_ops()

### ISSUE THREAD

**Issue by ChenyangLiu** [tensorflow/tensorflow#20991]
I worked on Tensorflow 1.8, and I used tf.add_check_numerics_ops() to check my model.
But it fails if I add tf.add_check_numerics_ops().
```
NotFoundError (see above for traceback): Unsuccessful TensorSliceReader constructor: Failed to find any matching files for model
	 [[Node: save/RestoreV2 = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
```

my code is here:
```
with tf.Session() as sess:
    logits, end_points = deeplab(inputs, reuse=False, is_training=False)
    sess.run(tf.global_variables_initializer())
    saver = tf.train.Saver()
    print 'Model established'
    saver.restore(sess, './models/xception65/model.ckpt')
    print 'Weight Loaded'
    checker = tf.add_check_numerics_ops()
    checker, logits_v = sess.run([checker, logits], feed_dict={inputs: img})
```

If I remove `checker` and `tf.add_check_numerics_ops()`, my code works fine.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by ChenyangLiu** [comment]
Sorry for lacking information.
OS : Ubuntu 14.04
Tensorflow: 1.8 installed from pip.
CUDA: 9
GPU model: GTX 1060 6GB
And as for the function `deeplab`, it's a warpper I wrote for deeplabv3+, which is available in tensorflow/models, using `slim` to establish model.

**Comment by ChenyangLiu** [comment]
Thanks for your help! I'll try to run check ops before constructing Saver.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @allenlavoie: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
